### PR TITLE
PPTP-325: liability weight page impl

### PR DESCRIPTION
**Pull request by tombart**

### CODE REVIEW

File: app/uk/gov/hmrc/plasticpackagingtax/registration/forms/LiabilityWeight.scala [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+case class LiabilityWeight(totalKg: Option[Long]) {}
+
+object LiabilityWeight {
+  implicit val format = Json.format[LiabilityWeight]
+
+  val maxTotalKg            = 100000000 // one hundred million
+  val minTotalKg            = 1000
+  val totalKg               = "totalKg"
+  val weightEmptyError      = "liabilityWeight.empty.error"
+  val weightOutOfRangeError = "liabilityWeight.outOfRange.error"
```
Comment (PauloPortugal) [inline app/uk/gov/hmrc/plasticpackagingtax/registration/forms/LiabilityWeight.scala:32]: I like this approach

File: app/uk/gov/hmrc/plasticpackagingtax/registration/views/registration_page.scala.html [hunk 2]
```diff
@@ -36,7 +36,7 @@
 
 @task(index: Int, title: String, linkText: String, linkCall: Call, status: TaskStatus) = {
     <h2 class="app-task-list__section">
-        <span class="app-task-list__section-number">@{index}</span>
+        <span class="app-task-list__section-number">@{index}.</span>
```
Comment (PauloPortugal) [inline app/uk/gov/hmrc/plasticpackagingtax/registration/views/registration_page.scala.html:39]: why do we need the `.` there?

File: build.sbt [hunk 3]
```diff
@@ -10,6 +10,7 @@ val silencerVersion = "1.7.0"
 
 lazy val microservice = Project(appName, file("."))
   .enablePlugins(play.sbt.PlayScala, SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin, SbtArtifactory, SbtWeb)
+  .disablePlugins(JUnitXmlReportPlugin)
```
Comment (PauloPortugal) [inline build.sbt:13]: Is this to remove that crappy alert?

**Review by PauloPortugal — APPROVED** [review]
Just one question about a `.` that should not be there... i guess.

File: build.sbt (hunk 3 above)
Comment (tombart) [reply build.sbt:13]: according to some confluence this should fix the error on jenkins where test results cannot be read

File: app/uk/gov/hmrc/plasticpackagingtax/registration/views/registration_page.scala.html (hunk 2 above)
Comment (PauloPortugal) [reply app/uk/gov/hmrc/plasticpackagingtax/registration/views/registration_page.scala.html:39]: For the `h2` titles

Comment (tombart) [reply app/uk/gov/hmrc/plasticpackagingtax/registration/views/registration_page.scala.html:39]: I think we're missing . on task list

**Pull request merged**